Sources/TSCBasic/FileSystem.swift

```diff
@@ -383,7 +383,11 @@ private class LocalFileSystem: FileSystem {
         try withTemporaryFile(dir: path.parentDirectory, deleteOnClose: false) { temp in
             do {
                 try writeFileContents(temp.path, bytes: bytes)
-                try FileManager.default.moveItem(atPath: temp.path.pathString, toPath: path.pathString)
```

Comment:
Maybe we should just switch to Foundation for atomic writes. It can be simplified to:

```swift
try bytes.withData {
    try $0.write(to: URL(fileURLWithPath: path.pathString), options: .atomic)
}
```

Reply:
Updated!